### PR TITLE
[ADD] project_categ_model

### PR DESCRIPTION
**Pull request by martintorresvm**
Add constrain and required field for categ_ids
	modified:   project_categ_model.py

### CODE REVIEW

**Comment by martintorresvm** [comment]
FYI @elicoidal

**Comment by martintorresvm** [comment]
Please review @elicoidal @dreispt @seb-elico

**Comment by elicoidal** [comment]
@victormartinelicocorp I think we should be able to have multiple tags.
What we need to remove is the mandatory attribute in the field tag.
For BR management, besides we need a specific field Category (m2o) on top of the tags linked to the same model and allowing only to display a subset.

**Comment by dreispt** [comment]
I tend to agree with Eric.

**Comment by martintorresvm** [comment]
Ok then I just remove the attribute required. And later I'll will propose a simple module to manage many2one categs.

Could you check again @elicoidal @dreispt ?

**Comment by martintorresvm** [comment]
> Ok then I just remove the attribute required. And later I'll will propose a simple module to manage many2one categs.
Could you check again @elicoidal @dreispt ?

@elicoidal I think this is done could you help to merge it?

**Review by dreispt — COMMENTED** [review]
Sorry @victormartinelicocorp , but this module is not doing any relevant changes to the module.

**Comment by elicoidal** [comment]
Lol indeed...
Somewhere we are lost in translation

**Comment by martintorresvm** [comment]
> @victormartinelicocorp I think we should be able to have multiple tags.
**What we need to remove is the mandatory attribute in the field tag.**
For BR management, besides we need a specific field Category (m2o) on top of the tags linked to the same model and allowing only to display a subset.

@elicoidal @dreispt If the point about **remove mandatory** not necessary I rather close this PR

Thanks for review

**Comment by dreispt** [comment]
@victormartinelicocorp I think you want to add the required attribute on the Task view, like you did to Issues on another PR.

**Comment by elicoidal** [comment]
I just need this "What we need to remove is the mandatory attribute in the field tag."
as we discussed long time ago.
Besides in task, add a specific field m2o linked to the category tags model

**Comment by dreispt** [comment]
I'm OK to remove the mandatory.
@victormartinelicocorp can we have a clean diff with just that change?

**Comment by martintorresvm** [comment]
> I just need this "What we need to remove is the mandatory attribute in the field tag."
as we discussed long time ago.
Besides in task, add a specific field m2o linked to the category tags model

> I'm OK to remove the mandatory.
@victormartinelicocorp can we have a clean diff with just that change?

Done, please have a check. @elicoidal @dreispt

File: project_categ/project_categ_view.xml [hunk 1]
```diff
@@ -44,10 +44,6 @@
                     <field name="task_categ_id" invisible="1"/>
                 </field>
 
-                <field name="categ_ids"  position="attributes">
-                    <attribute name="attrs">{'required':[('task_categ_id','!=',False)]}</attribute>
-                </field>
-
```
Comment (dreispt) [inline project_categ/project_categ_view.xml:50]: I expected this to be the only diff, no Python code changes.

**Comment by martintorresvm** [comment]
> I expected this to be the only diff, no Python code changes.

Sorry by that I made a mistake with Git reset.  Now I double check after commit and hope it's fine.

cc @elicoidal @dreispt

**Review by elicoidal — APPROVED** [review]

**Review by dreispt — APPROVED** [review]

**Pull request merged**